### ISSUE TITLE
Cocoapod podspec checksum updates

### ISSUE THREAD

**Issue by esteluk** [Instabug/Instabug-iOS#389]
Hey there,

We've encountered an issue where our builds have started to fail with this error:

```
[11:29:43]: ▸ Analyzing dependencies
[11:29:43]: ▸ Verifying no changes
[11:29:43]: ▸ [!] There were changes to the lockfile in deployment mode:
[11:29:43]: ▸ SPEC CHECKSUMS:
[11:29:43]: ▸ Instabug:
[11:29:43]: ▸ New Lockfile: 17c397e8dcaa93655c9ac112baddce937e36a0d0
[11:29:43]: ▸ Old Lockfile: 3fe272b0000a46f4a22ee1c19cc50d8fe0f9e784
[11:29:43]: Exit status of command 'bundle exec pod install --deployment --repo-update' was 1 instead of 0.
```

This appears to be because the cocoapods master spec has been [retrospectively updated](https://github.com/CocoaPods/Specs/commit/d767c576bf94ca19554f01c42eb5975aeb99650f#diff-347920df23f615369dfd8e58d1d8cfb0e8df9f76f518586271f4c2f0d1665e6d) for all previous releases to point to your self-hosted zip distribution, rather than distributing directly from this repository. 

This has a side-effect of causing most of our builds, including all our pre-existing release builds, to fail, particularly on CI, where the `--deployment` flag validates the that the checksums of the podspecs haven't been updated.

Obviously the cat is a bit out of the bag for this change, but is it possible to confirm that the changes to the pre-existing podspecs are expected? And if possible, it would be great if you could give a heads-up before similar breaking changes in the future!

**Comment by Kmohamed** [comment]
Hi @esteluk,
I completely understand your concern. We should announce this before proceeding with it. This something we will definitely handle in the future. So, I confirm that we changed all previous releases to point to your self-hosted zip distribution rather than distributing directly from our repository.

**Comment by Kmohamed** [comment]
We released `10.1` that maintains the old behavior and uses Github as a source for Cocoapods. Could you give it a try?

**Comment by Kmohamed** [comment]
Hi @esteluk, I think we fixed this issue in 10.1. Please feel free to reach out to us if you face any new issues.
